diff --git a/node_modules/@react-native-community/datetimepicker/android/build.gradle b/node_modules/@react-native-community/datetimepicker/android/build.gradle
index af810e8..d4bdf91 100644
--- a/node_modules/@react-native-community/datetimepicker/android/build.gradle
+++ b/node_modules/@react-native-community/datetimepicker/android/build.gradle
@@ -15,6 +15,7 @@ def isNewArchitectureEnabled() {
     // - Invoke gradle with `-newArchEnabled=true`
     // - Set an environment variable `ORG_GRADLE_PROJECT_newArchEnabled=true`
     return project.hasProperty("newArchEnabled") && project.newArchEnabled == "true"
+
 }
 
 apply plugin: 'com.android.library'
@@ -30,7 +31,8 @@ android {
       namespace "com.reactcommunity.rndatetimepicker"
     }
 
-    compileSdkVersion getExtOrIntegerDefault('compileSdkVersion')
+    // compileSdkVersion getExtOrIntegerDefault('compileSdkVersion')
+    compileSdkVersion = 34
 
     // Used to override the NDK path/version on internal CI or by allowing
     // users to customize the NDK path/version from their root project (e.g. for M1 support)
@@ -42,8 +44,10 @@ android {
     }
 
     defaultConfig {
-        minSdkVersion getExtOrIntegerDefault('minSdkVersion')
-        targetSdkVersion getExtOrIntegerDefault('targetSdkVersion')
+        // minSdkVersion getExtOrIntegerDefault('minSdkVersion')
+        // targetSdkVersion getExtOrIntegerDefault('targetSdkVersion')
+        minSdkVersion = 21
+        targetSdkVersion = 34
         buildConfigField "boolean", "IS_NEW_ARCHITECTURE_ENABLED", isNewArchitectureEnabled().toString()
     }
 
